platform/src/test/java/org/exoplatform/platform/qa/ui/platform/plf/PlfUnifiedSearchTestIT.java

```diff
@@ -408,4 +421,62 @@ public void test09_SearchDiscussions() {
     forumCategoryManagement.deleteCategory(nameCat);
 
   }
+
+  @Test
+  @Tag("ECMS-7784")
+  @BugInPLF("ECMS-7784")
```

Comment:
To be removed since this issue has been resolved.

Reply:
done

Reply:
There is still the tag @BugInPLF whereas the issue is fixed. Is it expected ?

Reply:
No, it is an oversight, I thought I removed it. 
Removed now.